rustls-bench/src/main.rs

```diff
@@ -1352,6 +1406,14 @@ impl TempBuffers {
     }
 }
 
+fn wall_time() -> f64 {
+    duration_nanos(
+        SystemTime::now()
```

Comment:
Shouldn't this be using `Instant` instead?

Reply:
I don't think you can go from an Instant to something that can be written a file? Aside from that, this is intended to produce similar output files to the C++ version which uses `gettimeofday` here.

Reply:
Well, you can get a `Duration` by comparing two `Instant`s but I guess that doesn't help if you're trying to match C++ output.